executor/ddl.go

```diff
@@ -104,6 +105,27 @@ func deleteTemporaryTableRecords(memData kv.MemBuffer, tblID int64) error {
 	return nil
 }
 
+func (e *DDLExec) getLocalTemporaryTables() *infoschema.LocalTemporaryTables {
```

Comment:
Why put it here but not `SessionVars`? I think it will probably be used in other modules except DDL.

Reply:
However, we don't want other modules to use `infoschema.LocalTemporaryTables` to ensure that there is only one place to get the schema.